scholarx/2022/index.html

```diff
@@ -464,15 +464,7 @@ <h1 class="display-3 text-center mb-5">FAQ for Mentees</h1>
                                         data-parent="#accordionExampleTwo"
                                 >
                                     <div class="card-body opacity-8 text-justify">
-                                        Mentor information and areas of expertise are
-                                        published on
-                                        the SEF website and social media. Students will be
-                                        asked
-                                        to pick their top three choices in mentors and will
-                                        be
-                                        assigned to selected mentors based on these choices
-                                        as well as the relative strength of their
-                                        application.
+                                        Mentor information and areas of expertise are published on the SEF ScholarX platform. Students will be asked to pick their choice of mentor and will be assigned to a mentor based on their choice as well as the relative strength of their application.
```

Comment:
Let's break this into few separate lines. This exceeds the max character limit.
Looks good otherwise. :)

Reply:
Sure!

Reply:
@Gravewalker666 Now is it ok?